### PR TITLE
add docker-compose and second level of DOMAIN_EXTENSION

### PR DESCRIPTION
**Pull request by nemmeviu**
These changes allow the project to be used for domains that have more than one hierarchy in DNS, and add docker-compose structure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,5 +31,19 @@ If you encounter problems, it might be interesting to follow the output of the c
 
 Now the freeradius should be up and running and accepting connections using the `user@example.com` and `user` as login names.
 
+## docker-compose
+
+After certificates copy to `./certs`.
```
Comment (hacor) [inline README.md:36]: After copying the certificates to `./certs`

File: README.md [hunk 2]
```diff
@@ -31,5 +31,19 @@ If you encounter problems, it might be interesting to follow the output of the c
...
+
+```
+cd env
+cp freeradius.env.sample freeradius.env
+```
+then, change the variables values for your environment and:
```
Comment (hacor) [inline README.md:42]: Change the variable values for your environment and run the following command

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+version: '3.2'
+services:
+  unifi-freeradius-ldap:
+    build:
+      dockerfile: Dockerfile
+      context: .
+    
+    # don't have second level support on BASE_DOMAIN
+    #image: haor/freeradius-google-ldap:latest
```
Comment (hacor) [inline docker-compose.yml:9]: I will rebuild this container after merging the PR

Comment (hacor) [reply docker-compose.yml:9]: You can then uncomment this line

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+version: '3.2'
+services:
+  unifi-freeradius-ldap:
+    build:
+      dockerfile: Dockerfile
+      context: .
+    
+    # don't have second level support on BASE_DOMAIN
```
Comment (hacor) [inline docker-compose.yml:8]: You can delete this comment, I will rebuild the container

File: env/freeradius.env.sample [hunk 5]
```diff
@@ -0,0 +1,5 @@
+ACCESS_ALLOWED_CIDR=ip.access.point
+BASE_DOMAIN=example
```
Comment (hacor) [inline env/freeradius.env.sample:2]: Add a comment line here with something like:
`# When setting up domain example.com.br`

File: env/freeradius.env.sample [hunk 6]
```diff
@@ -0,0 +1,5 @@
+ACCESS_ALLOWED_CIDR=ip.access.point
```
Comment (hacor) [inline env/freeradius.env.sample:1]: Add a comment line:
```
# Which IP range is allowed access to the freeradius controller. This should be the IP range of the access points talking to the freeradius container
ACCESS_ALLOWED_CIDR=192.168.1.0/24
```

**Review by hacor — CHANGES_REQUESTED** [review]
Thanks for the PR!
Seems good to me, only a few more changes concerning the documentation
If you have any questions, let me know!

**Comment by nemmeviu** [comment]
Ok. I will create a new PR with all changes.
Thanks!!

**Pull request closed without merging**